src/scss/vars/_grid.scss

```diff
@@ -1,5 +1,6 @@
 $grid-bp: (
   xs: 300px,
+  xss: 400px,
```

Comment:
```
xxs: 300px
xs: 400px
```
...would this have a negative impact apart from a find and replace job in the DS? Just think Medium, Small, Extra Small, Extra extra small - is a more logical sequence.